src/web/assets/cp/src/js/ElementEditor.js

```diff
@@ -2023,14 +2023,17 @@ Craft.ElementEditor = Garnish.Base.extend(
                   const $ul = $('<ul/>').appendTo(this.$activityContainer);
                   for (let i = 0; i < data.activity.length; i++) {
                     const activity = data.activity[i];
-                    const $li = $('<li/>', {
-                      tabindex: 0,
-                      'aria-label': activity.message,
-                    }).appendTo($ul);
+                    const $li = $('<li/>').appendTo($ul);
+                    const $button = $('<button/>', {
+                      type: 'button',
+                      class: 'activity-container__trigger',
+                      'aria-label': `${activity.userName} active, more info`,
```

Comment:
Couldn’t this be the same message as within the HUD? Then there’s no need to activate it.

Reply:
I think that might be confusing from a screen reader perspective, as a button signals that there's some sort of interactivity available. Having it function as a disclosure (with expanded/collapsed states, and with a label that signals that more info is available) solves that by having the screen reader experience be identical to the visual experience.